tests/unit/helpers/emberTest.js

```diff
@@ -0,0 +1,30 @@
+/*eslint-env node, mocha, es6 */
+'use strict';
+// var rewire = require('rewire');
+// var ember = rewire('../../../lib/helpers/ember.js');
+var ember = require('../../../lib/helpers/ember');
+var expect = require('chai').expect;
+
+describe('ember helper', function () {
+  // before(function () {
+  // });
+
+  it('returns a value', function () {
+    expect(ember).to.be.ok();
+  });
+
+  it('returns local by default on linux', function () {
+    if (process.platform !== 'win32') {
```

Comment:
Are you suppressing this test on windows because of the `spawn` you are calling? Would be good to have something for windows. But if `npm-run` is a feasible option that might improve that anyway.